src/OrderFormAcceptanceTests.Steps/Steps/CommonSteps.cs

```diff
@@ -99,6 +99,7 @@ public void GivenAnUnsubmittedOrderExists()
             order.SupplierName = "Really Kool Corporation";
 
             order.CommencementDate = new Faker().Date.Future();
+            order.CommencementDate = new DateTime(order.CommencementDate.Value.Year, order.CommencementDate.Value.Month, order.CommencementDate.Value.Day);
```

Comment:
Do we need the Faker line above if it's going to be rewritten via the DateTime function?

Reply:
yeah, so faker sets the date, but also the time. I want the time to be midnight, is there a way to set that via faker?